index.html

```diff
@@ -44,5 +45,153 @@ <h4>I'm a Software Developer</h4>
         </section>
     </section>
     
+    <section class="second-section">
+        <h3>My Recent Works</h3>
+        <hr />
+        <div class="recent-works"></div>
+        <article>
+            <h4 class="story-one">Multi-Post Stories</h4>
+            <p class="story-two">
+                A daily section of privately personalized reads;
+                no accounts or sign-ups required. Has been the industry's
+                standard dummy text ever since the 1500s, when an unknown 
+                printer took a standard dummy text.
+            </p>
+        </article>
+        <ul class="languages">
+            <li>css</li>
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>ruby</li>
+        </ul>
+        <button type="button" class="btn-project">See Project</button>
```

Comment:
- [ ] After fixing the styles I can see 👀 that your **See Project** button has a default color and not following the colors of the Figma design template 👇🏽. Kindly fix this by using the color as in the design 👍🏽.
### Figma design:
![image](https://user-images.githubusercontent.com/43172164/205329248-ca437393-f5f7-47a6-a01a-241ba27051e5.png)
### Your design:
![image](https://user-images.githubusercontent.com/43172164/205329364-bbde163e-53a3-4c62-b715-ae0a49182428.png)